Tests/Shared.Specs/EnumAssertionSpecs.cs

```diff
@@ -51,14 +51,15 @@ public void When_both_enums_are_equal_and_of_different_underlying_types_it_shoul
         public void When_both_enums_are_large_and_not_equal_it_should_throw()
         {
             // Arrange
```

Comment:
🤡 Consider updating the AAA style to match the rest.

Reply:
Please confirm if the comment is valid, this particular test seems to follow AAA style.

Reply:
I could imagine Dennis refers to surrounding the three section header comments with `//------------`

Reply:
Oh, right. Fixed.